docs/astro/src/content/docs/reference/window/window.mdx

```diff
@@ -57,3 +58,43 @@ Whether the window should be borderless/frameless or not.
 <SlintProperty propName="title" typeName="string">
 The window title that is shown in the title bar.
 </SlintProperty>
+
+## `MenuBar` element
+
+A `Window` can contain one `MenuBar` element. This special element is used to create a menu bar for that window.
+There can only be one `MenuBar` element in a `Window` and it must not be in a `for` or a `if`.
+The `MenuBar` doesn't have properties, but it should contain <Link type="MenuItem" /> as children that represent top level entries in the menu bar.
+Each `MenuItem` can contain further `MenuItem` as children that represent sub-menu
+
+Depending on the platform, the menu bar might be native or rendered by Slint.
+This means that for example, on MacOs, the menu bar be in top bar on the desktop where application usually have their top bar.
+The `width` and `height` property of the `Window` are the one of the client area, without counting the menu bar.
+The `x` and `y` properties of `Window` children are also relative to the client area.
```

Comment:
(The "at top of the screen" phrase is the term used in the Apple HIG, and it's short and precise)

```suggestion
This means that for example, on macOS, the menu bar will be at the top of the screen.
The `width` and `height` property of the `Window` define the client area, excluding the menu bar.
The `x` and `y` properties of `Window` children are also relative to the client area.
```